docs/DeveloperGuide.md

```diff
@@ -81,19 +82,33 @@ NotUs manages the flow of the application. On launch, it will create the necessa
   <br><em>Figure #</em>
 </p>
 
-#### <a id="parser"><ins>2.3 Parser</ins></a>
+#### <a id="parserManager"><ins>2.3 ParserManager</ins></a>
 
-The Parser class handles all the user input from the command terminal. It makes sense of the user input and calls the respective commands into action.
+The ParserManager manages the creation of specific parser objects based on the type of the command. The parser then
+ makes sense of the user input and calls the respective commands into action.
```

Comment:
```suggestion
The ParserManager manages the creation of specific parser objects based on the type of command. The parser then makes sense of the user input and calls the respective commands into action.
```

Reply:
This one is the ide auto ident to next line.